src/server.cpp

```diff
@@ -167,14 +167,26 @@ void ServicePort::open(uint16_t port)
 	pendingStart = false;
 
 	try {
+		IPAddress address;
```

Comment:
You could instantiate this as `any()` here instead of the else branch below. It's a trivial constructor.